packages/portal/spec-renderer/src/components/SpecDetails.vue

```diff
@@ -77,6 +80,8 @@ function showAndScrollToOperation() {
 
   swaggerRef.value.showOperation(props.activeOperation)
   swaggerRef.value.scrollToOperation(props.activeOperation)
+  // required for component testing
+  emit('active-operation')
```

Comment:
question: I see that this was added for the component tests, but I'm not sure I understand what the test is actually achieving by looking for this emit? 

Essentially the test is just verifying that the prop value is set to the correct value and that the component mounts (since this emit is triggered in the onMounted function), so I'm not sure this is actually providing any sort of test coverage beyond checking that the prop mounted and could be removed.